signing_service/signing_service/signing_service.py

```diff
@@ -96,9 +102,18 @@ def __init__(
         self.ethereum_private_key = ethereum_private_key  # type: str
         self.current_reconnect_delay: Union[int, None] = None
         self.was_sigterm_caught: bool = False
+        self.reconnection_counter = 0
+        self.maximum_reconnection_attempts = maximum_reconnect_attempts
 
         self._validate_arguments()
 
+        def _set_was_sigterm_caught_true(signum: int, frame: Optional[FrameType]) -> None:  # pylint: disable=unused-argument
```

Comment:
Why was this moved here?

Reply:
It was a part of a small refactor of `Signing Service` - -> you need to set it only once, and `run()` can be called multiple times, contrary to '__init__', so I figured out, that would be the better place for signal registration.

Reply:
It actually doesn't do anything before `run()` is called but ok. Please just fix indentation.